<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DADMClass</class>
 <widget class="QMainWindow" name="DADMClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DADM</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="reconstructionPushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reconstruction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ObliqueImagingButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Oblique Imaging</string>
    </property>
   </widget>
   <widget class="QPushButton" name="visualizationBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualization</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UpsamplingButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Upsampling</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport_from_file">
     <property name="title">
      <string>Import from file</string>
     </property>
     <addaction name="actionStructural_data"/>
     <addaction name="actionDiffusion_data"/>
    </widget>
    <addaction name="menuImport_from_file"/>
   </widget>
   <addaction name="menuFIle"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStructural_data">
   <property name="text">
    <string>Structural data</string>
   </property>
  </action>
  <action name="actionDiffusion_data">
   <property name="text">
    <string>Diffusion data</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dadm.qrc"/>
 </resources>
 <connections/>
</ui>
